src/componets/TabsTable.tsx

```diff
@@ -0,0 +1,39 @@
+import { Link } from 'react-router-dom';
```

Comment:
Here I understand a small typo, be careful with such mistakes. It should be 'src/`components`/TabsTable.tsx' instead of 'src/`componets`/TabsTable.tsx'.